iothub/service/src/IotHubServiceClient.cs

```diff
@@ -147,6 +147,12 @@ public IotHubServiceClient(string hostName, AzureSasCredential credential, IotHu
         /// <seealso href="https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-query-language"/>
         public QueryClient Query { get; protected set; }
 
+        /// <summary>
+        /// Subclient of <see cref="IotHubServiceClient"/> for Scheduled Jobs management.
+        /// </summary>
+        /// <seealso href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-jobs"/>
```

Comment:
Remember to remove the "en-us" from your URLs when including them.

```suggestion
        /// <seealso href="https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-jobs"/>
```